data/net/minecraft/client/renderer/texture/MissingTextureAtlasSprite.mapping

```diff
@@ -1,8 +1,15 @@
 CLASS net/minecraft/client/renderer/texture/MissingTextureAtlasSprite
+	METHOD <init> (Lnet/minecraft/client/renderer/texture/TextureAtlas;IIIII)V
+		ARG 1 atlas
+		ARG 2 mipLevel
```

Comment:
Conflicting name with `generateMipLevels` param name.

Reply:
These two also conflict in official names - the same constant is passed to `TextureAtlas.Preparations` and stored in the `mipLevel` field.

Reply:
Well yes. But I was saying that although it's passed there, you called it `mipLevels` in that method while `mipLevel` everywhere else. Apologies for not making that clear.